validators/crossingBuildings/map.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+var turf = require('turf');
+var _ = require('underscore');
+var rbush = require('rbush');
+
+module.exports = function(tileLayers, tile, writeData, done) {
+  var layer = tileLayers.osm.osm;
+  var buildings = {};
+  var bboxes = [];
+  layer.features.forEach(function(val) {
+    if (val.geometry.type === 'Polygon' && val.geometry.coordinates.length === 1 && val.properties.building === 'yes') {
+      var kinks = turf.kinks(val);
+      if (kinks.intersections.features.length === 0) {
+        val.properties._osmlint = 'crossingbuildings';
```

Comment:
Can you move adding `crossingbuildings` property right before writing to stdout? Doing it here doesn't make much sense logically.